tests/integration/runner/run.php

```diff
@@ -13,6 +13,7 @@
 $modulesToTest = [
     'gsitemap',
     'dashproducts',
+    'fakemodule',
```

Comment:
This is needed anytime we add a sample module. Maybe I should remove this hardcoded list and parse the module-samples folder instead ?

(maybe a new PR 😄 )